src/front/app/features/challenge/components/Challenge.js

```diff
@@ -74,6 +74,7 @@ class Challenge extends Component {
                             challenge={challenge}
                             onAccept={onAccept}
                             onComplete={onComplete}
+                            currentUser={userId}
```

Comment:
do we realy need to pass user id  ?
I tough that logic related to user id should stay in action creator